docs/dashboards/synchronize-components.md

```diff
@@ -130,6 +130,7 @@ Highlight sync can have additional options:
 sync: {
     highlight: {
         enabled: true,
+        affectedSeriesId: 'series-1',
```

Comment:
Please briefly describe this option.

Reply:
Just think, how about `relatedSeries` or `linkedSeriesId` names.

IMO `affected` series does not sound intuitive, however, it's only my impression.